test/spec/worker.spec.js

```diff
@@ -231,6 +230,38 @@ describe('Worker', function () {
         });
     });
 
+    describe('handle option parameters', () => {
+      let worker;
+      let initialArgs;
+
+      before(() => {
+        initialArgs = process.execArgv;
+      });
+
+      after(() => {
+        process.execArgv = initialArgs;
+      });
+
+      afterEach(() => {
+        worker.kill();
+      });
+
+      it('can override options', done => {
```

Comment:
I think we can simplify the tests a little. We are testing the behavior of `child_process.fork()` here now (`can override` test case).

I think the test does not need to cover the behavior of `child_process`. It should just test if the options are passed down to `child_process.fork()` and let it then just do its magic.

Agree?

Reply:
On the other hand... It will be more difficult to test whether passing down the options worked. Testing for the resulting `execArgv` is probably the better choice.

But I would still reduce it to just one test case. Maybe with one argument "inherited" by the parent process and one set using the options.

Reply:
@andywer  Yes I agree that it will be difficult to test if the options worked without checking execArgv. I removed the unnecessary test.

Reply:
With regards to adding one argument "inherited" by the parent process and one set using the options, I believe when we pass `options.execArgv` it overrides the parent/default parameters.